src/server-extension/resolvers/top_collections.ts

```diff
@@ -139,6 +143,7 @@ export class TopCollectionResolver {
                     .addSelect('NULLIF(MAX(sales_last_duration) OVER(), 0) AS max_sales')
                     .addSelect('sales_last_duration AS sales')
                     .addSelect('verified_at::text AS "verifiedAt"')
+                    .addSelect('created_at::text AS "createdAt"')
```

Comment:
This should either be `created_at` or the orderBy should be camelCase 
 ```suggestion
                    .addSelect('created_at::text AS "created_at"')
```